time: 2013-12-20 08:53:37.673895Z
tags: worker-0
test: tempest.scenario.test_volume_boot_pattern.TestVolumeBootPattern.test_volume_boot_pattern[compute,image,volume]
time: 2013-12-20 08:53:38.654796Z
failure: tempest.scenario.test_volume_boot_pattern.TestVolumeBootPattern.test_volume_boot_pattern[compute,image,volume] [ multipart
Content-Type: text/plain;charset="utf8"
pythonlogging:''
0
Content-Type: text/plain;charset="utf8"
stderr
0
Content-Type: text/plain;charset="utf8"
stdout
0
Content-Type: text/x-traceback;charset="utf8",language="python"
traceback
1A7
Traceback (most recent call last):
  File "/home/os1/tempest/tempest/scenario/test_volume_boot_pattern.py", line 189, in test_volume_boot_pattern
    volume_origin = self._create_volume_from_image()
  File "/home/os1/tempest/tempest/scenario/test_volume_boot_pattern.py", line 90, in _create_volume_from_image
    return self.create_volume(name=vol_name, imageRef=img_uuid)
NameError: global name 'img_uuid' is not defined
0
]
tags: -worker-0
